scripts/ipfs-pi-stream/install

```diff
@@ -24,3 +24,15 @@ sudo systemctl enable process-stream
 
 # Add hourly job to clear out old data
 echo "41 * * * * $USER /usr/local/bin/ipfs repo gc" | sudo tee --append /etc/crontab
+
+# Install the ipfs video player
+mkdir "$BASE_DIR/tmp"
+git clone https://github.com/tomeshnet/ipfs-live-streaming.git "$BASE_DIR/tmp/ipfs-live-streaming"
+VIDEO_PLAYER_PATH="$BASE_DIR/tmp/live-streaming/terraform/shared/video-player"
+sed -i s#__IPFS_GATEWAY_SELF__#/ipfs/# "$VIDEO_PLAYER_PATH/js/common.js"
+sed -i s#__IPFS_GATEWAY_ORIGIN__#https://ipfs.io/ipfs/# "$VIDEO_PLAYER_PATH/js/common.js"
+IPFS_ID=$(ipfs id | grep ID | head -n 1 | awk -F\" '{print $4}')
+sed -i "s#live.m3u8#/ipns/$IPFS_ID#" "$VIDEO_PLAYER_PATH/js/common.js"
```

Comment:
Shall we change https://github.com/tomeshnet/ipfs-live-streaming/blob/master/terraform/shared/video-player/js/common.js#L6 instead of messing up the previous line?

Reply:
Agreaed,

however because of  __IPFS_GATEWAY_ORIGIN__  (which is already replaced) and just the amount of lines it would take to do that i think its "safer" to keep as is.

Added pin repo instead.